tasks/rpmbuild-test

```diff
@@ -39,5 +57,7 @@ sed -i "/${fed_repo}_repo/d" manifest.txt
 echo "${fed_repo}_repo $(date --utc +%FT%T%Z)" >> manifest.txt
 sort manifest.txt -o manifest.txt
 rsync --delete -stats -a manifest.txt fedora-atomic@artifacts.ci.centos.org::fedora-atomic/${RSYNC_BRANCH}/repo
+# Delete the rsync lock we placed
+rsync -vr --delete $(mktemp -d)/ fedora-atomic@artifacts.ci.centos.org::fedora-atomic/${RSYNC_BRANCH}/repo/lockdir/
```

Comment:
this should be done in the trap handler so that we always remove the lock even if we abort for some reason.